src/requesthandler/RequestHandler_Inputs.cpp

```diff
@@ -549,6 +549,7 @@ RequestResult RequestHandler::SetInputAudioSyncOffset(const Request& request)
  * Gets the audio monitor type of an input.
  *
  * The available audio monitor types are:
+ *
```

Comment:
Can you go over the rest of the documentation and make sure it's formatted correctly?

Reply:
Sorry, will finally get back to this now.
Probably a few updates needed here and there but should be quick overall.